stdlib/configparser.pyi

```diff
@@ -5,7 +5,31 @@ from re import Pattern
 from typing import Any, ClassVar, Literal, TypeVar, overload
 from typing_extensions import TypeAlias
 
-if sys.version_info >= (3, 12):
+if sys.version_info >= (3, 13):
+    __all__ = (
+        "NoSectionError",
+        "DuplicateOptionError",
+        "DuplicateSectionError",
+        "NoOptionError",
+        "InterpolationError",
+        "InterpolationDepthError",
+        "InterpolationMissingOptionError",
+        "InterpolationSyntaxError",
+        "ParsingError",
+        "MissingSectionHeaderError",
+        "ConfigParser",
+        "RawConfigParser",
+        "Interpolation",
+        "BasicInterpolation",
+        "ExtendedInterpolation",
+        "SectionProxy",
+        "ConverterMapping",
+        "DEFAULTSECT",
+        "MAX_INTERPOLATION_DEPTH",
+        "UNNAMED_SECTION",
+        "MultilineContinuationError",
```

Comment:
You could shorten this by adding something like `if sys.version_info >= (3, 13): __all__ += ["MultilineContinuationError"]`. We can also get rid of the existing duplication of `__all__` this way.

Reply:
It's not possible to get rid of the existing duplication without stubtest complaining because it's a tuple in one branch but a list in the other branch

Reply:
Yeah after 3.12 this is a tuple now, and in 3.13 there are a few additions and removals which makes editing it not simple

Reply:
We could do something like this to make it more concise, but I personally think the nested `sys.version_info` branches are a bit confusing:

```diff
--- a/stdlib/configparser.pyi
+++ b/stdlib/configparser.pyi
@@ -5,32 +5,8 @@ from re import Pattern
 from typing import Any, ClassVar, Literal, TypeVar, overload
 from typing_extensions import TypeAlias
 
-if sys.version_info >= (3, 13):
-    __all__ = (
-        "NoSectionError",
-        "DuplicateOptionError",
-        "DuplicateSectionError",
-        "NoOptionError",
-        "InterpolationError",
-        "InterpolationDepthError",
-        "InterpolationMissingOptionError",
-        "InterpolationSyntaxError",
-        "ParsingError",
-        "MissingSectionHeaderError",
-        "ConfigParser",
-        "RawConfigParser",
-        "Interpolation",
-        "BasicInterpolation",
-        "ExtendedInterpolation",
-        "SectionProxy",
-        "ConverterMapping",
-        "DEFAULTSECT",
-        "MAX_INTERPOLATION_DEPTH",
-        "UNNAMED_SECTION",
-        "MultilineContinuationError",
-    )
-elif sys.version_info >= (3, 12):
-    __all__ = (
+if sys.version_info >= (3, 12):
+    __all__: tuple[str, ...] = (
         "NoSectionError",
         "DuplicateOptionError",
         "DuplicateSectionError",
@@ -46,12 +22,15 @@ elif sys.version_info >= (3, 12):
         "Interpolation",
         "BasicInterpolation",
         "ExtendedInterpolation",
-        "LegacyInterpolation",
         "SectionProxy",
         "ConverterMapping",
         "DEFAULTSECT",
         "MAX_INTERPOLATION_DEPTH",
     )
+    if sys.version_info >= (3, 13):
+        __all__ += ("UNNAMED_SECTION", "MultilineContinuationError")
+    else:
+        __all__ += ("LegacyInterpolation",)
 else:
     __all__ = [
```

Though it's verbose, I'd go with what this PR currently has for this module.

Reply:
Oh I didn't notice the list/tuple difference. It's OK to go with what we currently have then.